doc/guide/build_windows.hpp

```diff
@@ -10,11 +12,41 @@ This document discusses how to build mlpack for Windows from source, so you can
 later create your own C++ applications.  There are a couple of other tutorials
 for Windows, but they may be out of date:
 
- * <a href="https://github.com/mlpack/mlpack/wiki/WindowsBuild">Github wiki Windows Build page</a>
- * <a href="http://keon.io/mlpack-on-windows">Keon's tutorial for mlpack 2.0.3</a>
- * <a href="https://overdosedblog.wordpress.com/2016/08/15/once_again/">Kirizaki's tutorial for mlpack 2</a>
+ * <a href="https://github.com/mlpack/mlpack/wiki/WindowsBuild">Github wiki Windows Build page</a><br/>
+ * <a href="http://keon.io/mlpack-on-windows">Keon's tutorial for mlpack 2.0.3</a><br/>
+ * <a href="https://overdosedblog.wordpress.com/2016/08/15/once_again/">Kirizaki's tutorial for mlpack 2</a><br/>
 
-Those guides could be used in addition to this tutorial.
+Those guides could be used in addition to this tutorial. Furthermore, mlpack is
+now available for Windows installation through vcpkg:
+
+- Install Git (https://git-scm.com/downloads and execute setup)
+
+- Install CMake (https://cmake.org/ and execute setup)
+
+- Install vcpkg (execute in PowerShell):
+
+@code
+> git clone https://github.com/Microsoft/vcpkg.git
```

Comment:
Not sure we should list the install routine for vcpkg here; does the PowerShell come with git?

Reply:
Agreed with that---we can point readers to vcpkg's documentation for that.

Reply:
Okay, updated accordingly. Thanks.